benchmark/benchmark.cpp

```diff
@@ -262,7 +264,7 @@ int Benchmark<T>::check_params()
         }
     }
 
-    size_t wordsize_limit = _log2<T>(n) + 1;
+    size_t wordsize_limit = nttec::arith::_log2<T>(n) + 1;
```

Comment:
Now we have proper namespaces we can now remove underscores from arith functions (as they were used to distinguish them mainly from classes methods).

Reply:
Yes, that's another patch I already have
I didn't put it in this PR because I think it was more related to #156.
That being said, arith is not a class so it makes more sense to add it in this PR.